beta/aws-ecs-apigateway-cfn/README.md

```diff
@@ -0,0 +1,132 @@
+# Deploy 1Password SCIM Bridge with AWS CloudFormation (API Gateway)
+
+_Learn how to deploy 1Password SCIM Bridge using [AWS CloudFormation](https://docs.aws.amazon.com/cloudformation/index.html)._
+
+This guide can be used to deploy 1Password SCIM Bridge as a [stack](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-whatis-concepts.html#cfn-concepts-stacks) using the [AWS CloudFormation console](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-using-console.html).
+
+The included [AWS CloudFormation template](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stack-templates-overview.html) is a working example that is suitable for use in a production environment. Some common customizations are included for convenience, but it is intentionally minimal for simplicity, to allow any identity provider to connect to its public endpoint, and to facilitate its use as a base for a custom deployment that meets your specific requirements.
+
+This template deploys 1Password SCIM Bridge and the required Redis cache as Fargate tasks within ECS services integrated with AWS Cloud Map for service discovery. A VPC with at least two public subnets is required; these will be created automatically if not provided. Credentials are stored in and provided by AWS Secrets Manager. A public TLS endpoint is provided by Amazon API Gateway. Minimal IAM roles, policy, and security group resources are included for access control and principle of least privelege.
+
+## Prerequisites
+
+- A 1Password account with an active 1Password Business subscription or trial
+> [!TIP]
+> Try 1Password Business **free** for 14 days: <https://start.1password.com/sign-up/business>
+- An AWS account with the permissions and available quota to create and manage the described resources
+
+## Before you begin
+
+Consult the [Preparation Guide](/PREPARATION.md) in this repository. Since this CloudFormation template will create all necessary resources, it does not use the CertificateManager component to acquire and manage a TLS certificate. API Gateway provides the TLS endpoint. No DNS records are required.
+
+1. Download the [`op-scim-bridge.yaml`](./op-scim-bridge.yaml) template file from this repository to your computer.
+2. Follow the steps in [Automate provisioning in 1Password Business using SCIM](https://support.1password.com/scim/#step-1-set-up-and-deploy-1password-scim-bridge) to generate credentials for your SCIM bridge.
+3. Save the `scimsession` credentials file and the associated bearer token as items in your 1Password account.
+4. Download the `scimsession` file to your computer.
+
+## Step 1: Upload the stack template file
+
+1. Sign in to AWS and [create a stack](https://console.aws.amazon.com/cloudformation/home#/stacks/create).
```

Comment:
Can we add the instructions for if you don't want to click the link?

Reply:
As in using terminal commands?